lib/msf/core/db_manager/http/servlet_helper.rb

```diff
@@ -13,21 +13,22 @@ def set_error_on_response(error)
   end
 
   def set_empty_response
-    [200,  '']
+    set_json_data_response(response: '')
   end
 
   def set_json_response(data, includes = nil, code = 200)
-    headers = {'Content-Type' => 'application/json'}
+    headers = { 'Content-Type' => 'application/json' }
     [code, headers, to_json(data, includes)]
   end
 
   def set_json_data_response(response:, includes: nil, code: 200)
-    data_response = {"data": response}
+    data_response = { "data": response }
```

Comment:
Minor: (Yes I originally added that code!) use single quotes around data for string consistency in this module.

Reply:
Using the `key: value` format always puts the key as a symbol. So unless it matters, I'll just remove the quotes altogether.

```
2.5.1 :001 > h = { 'quotes': 1, symbol: 2, "quotearrow" => 3 }
 => {:quotes=>1, :symbol=>2, "quotearrow"=>3} 
```

Reply:
Even better point! I'm not sure how I ended up with strings originally. I must have been working in reverse from actual JSON.